toplevel/topdirs.ml

```diff
@@ -435,18 +435,6 @@ let is_nonrec_type id td =
   | true, _ | _, false -> Trec_first
     (* note: true, true is possible *)
 
-let () =
-  reg_show_prim "show_type"
-    (fun env loc id lid ->
-       let path, desc = Env.lookup_type ~loc lid env in
-       let id, rs = match path with
-         | Pident id -> id, is_nonrec_type id desc
-         | _ -> id, Trec_first
-       in
-       [ Sig_type (id, desc, rs, Exported) ]
-    )
-    "Print the signature of the corresponding type constructor."
-
```

Comment:
Sorry, I didn't mean to change the current order of sub `show` command, you should leave `show_type` at this position.

Reply:
fixed in a7e9da1